src/main/java/meteordevelopment/meteorclient/systems/modules/misc/PacketCanceller.java

```diff
@@ -48,4 +56,8 @@ private void onReceivePacket(PacketEvent.Receive event) {
     private void onSendPacket(PacketEvent.Send event) {
         if (c2sPackets.get().contains(event.packet.getClass())) event.cancel();
     }
+
+    public boolean silentDisconnect() {
+        return silentDisconnect.get();
```

Comment:
wouldn't it be better to put an `isActive()` check in here, considering the code accessing this does not?